tools/ci/license_check.sh

```diff
@@ -18,15 +18,11 @@
 ################################################################################
 
 MVN_CLEAN_COMPILE_OUT=$1
```

Comment:
```suggestion
if [[ "$#" != "2" ]]; then
  echo "Usage: $0 <maven-clean-compile-output> <deployed-artifacts-folder>"
  echo "    <maven-clean-compile-output>                 A file containing the Maven run's log output."
  echo "    <deployed-artifacts-folder>                        A mirror folder that contains the deployed Flink artifacts."
  echo ""
  echo "The environment variable MVN is used to specify the Maven binaries."
  echo "See further details in the JavaDoc of LicenseChecker.".
  exit 1
fi

MVN_CLEAN_COMPILE_OUT=$1
```
can we add a usage info in case a user wants to call a script individually? It serves as some kind of documentation for this specific script. WDYT?

Reply:
The same would apply to `tools/ci/verify_bundled_optional.sh` and `tools/ci/verify_scala_suffixes.sh`